### PR TITLE
Chart store header

### PR DESCRIPTION
**Pull request by shivani170**
# Description

Idea is to make all the Chart store headers  uniform

1. chart store
Screenshot 2022-06-03 at 5 18 13 PM

2. Chart store > Chart Group
Screenshot 2022-06-03 at 5 18 49 PM

3. Chart store > Multiple chart Group
Screenshot 2022-06-03 at 5 20 06 PM

4. Chart Details
Screenshot 2022-06-03 at 5 22 22 PM

5. Chart Edit
Screenshot 2022-06-03 at 5 20 31 PM

6. Create values
Screenshot 2022-06-03 at 5 21 02 PM

Fixes https://github.com/devtron-labs/devtron/issues/1787

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/charts/MultiChartSummary.tsx [hunk 1]
```diff
@@ -1,200 +1,328 @@
...
-                        toggleChart={toggleChart && configureChartIndex !== index ? e => toggleChart(index) : null}
-                        chart={chart}
-                        selected={configureChartIndex === index}
-                        getChartVersionsAndValues={getChartVersionsAndValues ? () => getChartVersionsAndValues(chart.id, index) : null}
-                        hideDeployedValues={hideDeployedValues}
-                    />
-                )
-                }
-            </div>}
+            {chartListing && (
```
Comment (vivek-devtron) [inline src/components/charts/MultiChartSummary.tsx:35]: Can we wrap all these in single condition?

File: src/components/charts/chartValues/ChartValues.tsx [hunk 2]
```diff
@@ -145,121 +162,159 @@ export default class ChartValues extends Component<ChartValuesProps, ChartValues
...
+            <div className="flex left">
+                <BreadCrumb breadcrumbs={breadcrumbs} />/ Create Custom Values
             </div>
-            <div className="page-header__cta-container"/>
-        </div>
+        )
+    }
+    return (
+        <PageHeader isBreadcrumbs={true} breadCrumbs={renderChartValueBreadcrumbs} />
+        // <div className="page-header">
```
Comment (vivek-devtron) [inline src/components/charts/chartValues/ChartValues.tsx:315]: Remove commented code

File: src/components/charts/MultiChartSummary.tsx (hunk 1 above)
Comment (shivani170) [reply src/components/charts/MultiChartSummary.tsx:35]: Done

File: src/components/charts/chartValues/ChartValues.tsx (hunk 2 above)
Comment (shivani170) [reply src/components/charts/chartValues/ChartValues.tsx]: doen

**Review by vivek-devtron — APPROVED** [review]

**Review by sohel-rp — APPROVED** [review]

**Pull request merged**